nilearn/decomposition/multi_pca.py

```diff
@@ -316,8 +330,12 @@ def fit(self, imgs, y=None, confounds=None):
         else:
             data = subject_pcas[0]
             variance = subject_svd_vals[0]
+            if self.keep_data_mem:
+                subject_datas = subject_datas[0]
```

Comment:
"Data are", not "data is" -> therefore datas might be a bad name for a variable?